server/test/services/export/JsonExporterServiceTest.java

```diff
@@ -958,18 +1024,96 @@ public void export_whenFileUploadQuestionIsNotAnswered_valueInResponseIsNull() {
         exporter.export(
             fakeProgram.getProgramDefinition(),
             IdentifierBasedPaginationSpec.MAX_PAGE_SIZE_SPEC_LONG,
-            SubmittedApplicationFilter.EMPTY);
+            SubmittedApplicationFilter.EMPTY,
+            false);
     ResultAsserter resultAsserter = new ResultAsserter(resultJsonString);
 
     resultAsserter.assertJsonAtApplicationPath(
         ".applicant_file",
         """
         {
           "file_key" : null,
+          "file_urls" : [ ],
           "question_type" : "FILE_UPLOAD"
         }""");
   }
 
+  @Test
+  public void
+      export_whenFileUploadQuestionIsAnswered_whenMultipleFileUploadDisabled_fileKeyListEqualsFileKey() {
```

Comment:
should we include the `whenMultipleFileUploadDisabled/Enabled` in the other test names? It makes it clearer which scenario is being tested but also makes the names petty long...

Reply:
My thought here (or actually it was a reviewer's thought in a PR I've been trying to continue :) was to rename old tests that are going to be deleted with something like "whenMultipleFileUploadDisabled", but keeping the newer copies (where the feature enabled) of those tests un-changed with the simpler name. Then when you launch the feature, you go and delete any "whenMultipleFileDisabled" named tests, and leave the others unchanged. That way you don't end up with legacy "WhenMultipleFileUpload" names long after that feature isn't relevant since all file uploads are multiple.

I don't have strong feelings either way, but I've mostly been trying to do this for my unit tests since that PR.

Reply:
Ah makes sense to me!